client-registry-jempi/docker-compose.kafdrop-dev.yml

```diff
@@ -1,8 +0,0 @@
-version: '3.9'
-
-services:
-  jempi-kafdrop:
```

Comment:
we need to remove the config importer in the importer folder as well (kafka.js and docker-compose.config.yml)